# Scientific Large Language Model (LLM)

This repository collects some scientific LLM papers.

<!-- >What can **Large Language Models (LLMs)** do for Protein? 

ðŸ™Œ This repository collects some LLM-based protein papers.

ðŸ˜Ž Welcome to recommend missing papers through **`Adding Issues`** or **`Pull Requests`**.  -->

<!-- ## ðŸ”” News
- **2023-07  We create this repository to maintain a paper list on *Large Language Models* appiled in *Protein*.**

*Todo:*
1. - [ ] `Fine-grained classification of papers`
2. - [ ] `Update paper project / code` -->

## Content
- [Scientific Large Language Model (LLM)](#scientific-large-language-model-llm)
  - [Content](#content)
  - [Text LLM Papers](#text-llm-papers)
  - [Protein LLM Papers](#protein-llm-papers)
    - [Protein sequence generation](#protein-sequence-generation)
    - [Protein function prediction](#protein-function-prediction)
  - [Molecule LLM Papers](#molecule-llm-papers)
  - [Genome LLM Papers](#genome-llm-papers)
  - [Multimodal LLM Papers](#multimodal-llm-papers)
  - [Contribution](#contribution)
    - [ðŸ‘¥ Contributors](#-contributors)

---

## Text LLM Papers
### General model
- `2018` Improving Language Understanding by Generative Pre-Training,[OpenAI](https://cdn.openai.com/research-covers/language-unsupervised/language_understanding_paper.pdf)
- `2019`Language Models are Unsupervised Multitask Learner,[OpenAI](https://cdn.openai.com/better-language-models/language_models_are_unsupervised_multitask_learners.pdf)
- `2019` BERT: Pre-training of Deep Bidirectional Transformers for Language Understanding,[NAACL](https://aclanthology.org/N19-1423/)
- `2020` Language Models are Few-Shot Learners,[arXive](https://arxiv.org/abs/2005.14165)

### Scientific model
- `2019` SciBERT: A Pretrained Language Model for Scientific Text,[arXiv](https://arxiv.org/abs/1903.10676)
- `2023` Large language models encode clinical knowledge,[nature](https://www.nature.com/articles/s41586-023-06291-2)

## Protein LLM Papers
<!-- è¯·ä»¿ç…§æ­¤æ ¼å¼ï¼Œæœ€å¥½èƒ½å¯¹æ–‡ç« åˆ†ç±»ï¼Œç„¶åŽæŒ‰æ—¶é—´é¡ºåºæ·»åŠ -->
###  Protein sequence generation
- `2020` ProGen: Language Modeling for Protein Generationï¼Œ[arXiv](https://doi.org/10.48550/arXiv.2004.03497)
- `2022` Large language models generate functional protein sequences across diverse families, [Nature Biotechnology]( https://doi.org/10.1038/s41587-022-01618-2)
- 
###  Protein function prediction
- `2019` A High Efficient Biological Language Model for Predicting Proteinâ€“Protein Interactions, [Cells](https://doi.org/10.3390/cells8020122)
- `2020` TRANSFORMER PROTEIN LANGUAGE MODELS ARE UNSUPERVISED STRUCTURE LEARNERS, [bioRxiv](https://doi.org/10.1101/2020.12.15.422761)
- `2021` Highly accurate protein structure prediction with AlphaFold, [nature](https://doi.org/10.1038/s41586-021-03819-2)
- `2022` Language models of protein sequences at the scale of evolution enable accurate structure prediction, [bioRxiv](https://doi.org/10.1101/2022.07.20.500902)
- `2022` Accurate prediction of protein structures and interactions using a 3-track neural network, [Science](https://doi.org/10.1126/science.abj8754)
- 

## Molecule LLM Papers
- 

## Genome LLM Papers
- 

## Multimodal LLM Papers
### Text multimodal model
### Protein multimodal model
- `2023` Enhancing the Protein Tertiary Structure Prediction by Multiple Sequence Alignment Generation, [arXiv](https://arxiv.org/abs/2306.01824)
- `2022` Evolutionary-scale prediction of atomic level protein structure with a language model, [bioRxiv](https://www.biorxiv.org/content/10.1101/2022.07.20.500902v3)
### molecular multimodal model
- `2023` GIT-Mol: A Multi-modal Large Language Model for Molecular Science with Graph, Image, and Text, [arXiv](https://arxiv.org/abs/2308.06911)
- `2023` MolFM: A Multimodal Molecular Foundation Model, [arXiv](https://arxiv.org/abs/2307.09484)
- `2023` Empowering Molecule Discovery for Molecule-Caption Translation with Large Language Models: A ChatGPT Perspective, [arXiv](https://arxiv.org/abs/2306.06615)
- `2022` Translation between Molecules and Natural Language, [arXiv](https://arxiv.org/abs/2204.11817)
- `2022` Molecular contrastive learning of representations via graph neural networks, [Nature](https://www.nature.com/articles/s41467-022-28494-3)
- `2022` A deep-learning system bridging molecule structure and biomedical text with comprehension comparable to human professionals, [Nature](https://www.nature.com/articles/s42256-022-00447-x)
- `2021` Text2Mol: Cross-Modal Molecule Retrieval with Natural Language
Queries, [EMNLP](https://aclanthology.org/2021.emnlp-main.47/)
### Gene multimodal model
- `2023` PCell-type-specific prediction of 3D chromatin organization enables high-throughput in silico genetic screening, [Nature](https://www.nature.com/articles/s41587-022-01612-8)
- `2022` Pan-cancer integrative histology-genomic analysis via multimodal deep learning, [Cancer Cell](https://www.cell.com/cancer-cell/fulltext/S1535-6108(22)00317-8)


## Contribution
### ðŸ‘¥ Contributors


<!-- ### ðŸŽ‰ Contributing ( welcome ! )

- âœ¨ Add a new paper or update an existing Protein-related LLM paper.
- ðŸ§ Use the same format as existing entries to describe the work.
- ðŸ˜„ A very brief explanation why you think a paper should be added or updated is recommended (Not Neccessary) via **`Adding Issues`** or **`Pull Requests`**.

**Don't worry if you put something wrong, they will be fixed for you. Just feel free to contribute and promote your awesome work here! ðŸ¤© We'll get back to you in time ~ ðŸ˜‰** -->


